swap/src/protocol/alice.rs

```diff
@@ -215,6 +215,7 @@ impl Builder {
     }
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
I am leaving this for now because it stems from sending a psbt instead of a transaction in message0.

Reply:
> I am leaving this for now because it stems from sending a psbt instead of a transaction in message0.

I do not understand your justification. Boxing it is very easy.

Reply:
To be fair, a PSBT is probably the nicer solution anyway because that is the format designed for collaboratively signing a transaction.